Readability.js

```diff
@@ -469,9 +476,9 @@ Readability.prototype = {
 
       if (totalCount === 0 && !this._getInnerText(paragraph, false))
         paragraph.parentNode.removeChild(paragraph);
-    });
+    }, true);
 
-    this._forEachNode(articleContent.getElementsByTagName("br"), function(br) {
+    this._forEachNode(this._getAllNodesWithTag(articleContent, ["br"]), function(br) {
```

Comment:
Yes, but algorithm of deleting `<br>`s depends on order,  because it uses `nextSiblings` and do some magic with them, so I decided to live forward direction here.